# ==============================================================================
# REGRESSOR COMMIT DIFF
# ==============================================================================
# File: accessible/ipc/other/PDocAccessible.ipdl
# Commit: 5e92b9c7d479
# Full Hash: 5e92b9c7d4794af1d7ab0fb5c278c2e681a18fc6
# Author: Eitan Isaacson <eitan@monotonous.org>
# Date: 2023-01-28 21:11:06
# Regressor Bug: 1811092
# File Overlap Count: 1
# Description:
#   Bug 1811092 - P1: Add new async add/set text selection message. r=Jamie
#   
#   Differential Revision: https://phabricator.services.mozilla.com/D167201
# ==============================================================================

diff -r 7cebaa85f4ff -r 5e92b9c7d479 accessible/ipc/other/PDocAccessible.ipdl
--- a/accessible/ipc/other/PDocAccessible.ipdl	Fri Jan 27 20:03:06 2023 +0000
+++ b/accessible/ipc/other/PDocAccessible.ipdl	Fri Jan 27 20:15:11 2023 +0000
@@ -224,6 +224,10 @@
   [Nested=inside_sync] sync RemoveFromSelection(uint64_t aID, int32_t aSelectionNum)
     returns(bool aSucceeded);
 
+  async SetTextSelection(uint64_t aStartID, int32_t aStartOffset,
+                     uint64_t aEndID, int32_t aEndOffset,
+                     int32_t aSelectionNum);
+
   async ScrollSubstringTo(uint64_t aID, int32_t aStartOffset, int32_t aEndOffset,
                           uint32_t aScrollType);
   async ScrollSubstringToPoint(uint64_t aID,